pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -68,7 +69,9 @@ func NewTokensController(cl clientset.Interface, options TokensControllerOptions
 		token:  options.TokenGenerator,
 		rootCA: options.RootCA,
 	}
-
+	if cl != nil && cl.Core() != nil && cl.Core().GetRESTClient() != nil {
+		metrics.RegisterMetricAndTrackRateLimitierUsage("serviceaccount_controller", cl.Core().GetRESTClient().GetRateLimiter())
```

Comment:
Somewhere, you need a test that the client.GetRESTClient().GetRateLimiter() call sequence actually gets the right rate limiter. I can see that getting broken in the future (by e.g. someone modifying a client's RESTClient or something)